libs/wingc/turbo.json

```diff
@@ -3,16 +3,17 @@
   "extends": ["//"],
   "pipeline": {
     "compile": {
+      "dependsOn": ["@winglang/wingii#compile", "@winglang/tree-sitter-wing#compile"],
```

Comment:
wingc compile does not depend on `^compile` because it does not need `@winglang/sdk#compile`